src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/IHtmlGenerator.cs

```diff
@@ -157,6 +157,7 @@ public interface IHtmlGenerator
             ModelExplorer modelExplorer,
             string expression,
             object value,
+            string format,
```

Comment:
@Eilon please reconfirm the addition of this parameter in the `public` but rarely-used-by-others `IHtmlGenerator` interface is :ok: for 1.1.0.